lib/components/user/nav-login-button.css

```diff
@@ -25,6 +25,18 @@
   background-color: #080808;
 }
 
+p {
+  padding: 5px 15px;
+  margin: 0;
+  font-weight: 600;
+  font-size: 16px;
+}
+
+ul {
+  list-style: none;
+  padding-inline-start: 0;
+}
+
```

Comment:
These are being applied to ALL `p` and `ul` elements site-wide, can you add some specificity to the selector, so it's only being applied to this menu?